### DISCUSSION TITLE
posts/javascript-proxy

### DISCUSSION THREAD

**Discussion by giscus** [witch-factory/witch-next-blog#17] [General]
# posts/javascript-proxy

ES6에서 도입된 Proxy, Reflect에 대해 알아보자

https://witch.work/posts/javascript-proxy

**Comment by cog25** [comment]
안녕하세요! 글 잘 읽었습니다 :)

다만 본문의 [프록시의 한계점](https://witch.work/posts/javascript-proxy#%ED%94%84%EB%A1%9D%EC%8B%9C%EC%9D%98-%ED%95%9C%EA%B3%84%EC%A0%90) 부분에서 마지막 문장이 끝나지 않아, 오류일 것 같아 코멘트를 남깁니다. 확인 가능하실까요?

**Reply by witch-factory** [reply]
내용을 쓰다가 그냥 올려 버렸네요. 조만간 보충하겠습니다. 알려주셔서 감사합니다!
